tests/torch2/function_hook/quantization/strip/test_strip_dequantize.py

```diff
@@ -155,7 +170,7 @@ def test_sym_fq_to_decompressor(param: ParamSymFQ):
 
     scale_shape = (1, 1)
     scale = torch.tensor(SCALE_SAMPLE)
-    scale = scale.expand(scale_shape).to(torch.float16)
+    scale = scale.expand(scale_shape)
```

Comment:
Doesn't influence the result, just more aligned with default precision in FQ (float32).